redirect/example/main.tf

```diff
@@ -0,0 +1,78 @@
+provider "aws" {
+  region = "eu-west-1" # Ireland
+}
+
+provider "aws" {
+  alias  = "acm"
+  region = "us-east-1"
```

Comment:
you might want to just create everything in us-east-1, will make the example simpler